dcap/types/Cargo.toml

```diff
@@ -28,7 +28,7 @@ p256 = { version = "0.13.0", default-features = false, features = ["ecdsa-core",
 serde = { version = "1.0", default-features = false, features = ["derive"] }
 sha2 = { version = "0.10.6", default-features = false }
 static_assertions = "1.1.0"
-subtle = { version = "2.4.1", default-features = false }
+subtle = { version = "2", default-features = false }
```

Comment:
Limited this to the Major version as specifying the patch version can cause a bit of grief with dependents like https://github.com/mobilecoinfoundation/mobilecoin.
I want to find the time to do a minimum version build, but it's hard to justify the investment right now.